src/app/diary-module/containers/diary-home/diary-home.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core'
+import { FormControl } from '@angular/forms'
+
+@Component({
+    selector: 'app-diary-home',
+    templateUrl: './diary-home.component.html',
+    styleUrls: ['./diary-home.component.scss'],
+})
+export class DiaryHomeComponent implements OnInit {
+    title = new FormControl('')
+    description = new FormControl('')
+    expand = new FormControl('collapse')
+
+    constructor() {}
+
+    ngOnInit(): void {
+        const handleClickOutside = (e: any) => {
+            if (e.target.id === 'title' || e.target.id === 'description') return
```

Comment:
follow this https://plainenglish.io/blog/creating-an-angular-directive-to-detect-clicking-outside-an-object-afd6c07212ef